### PR TITLE
cmd/tailscale/cli: improve tailnet lock help

### PR DESCRIPTION
**Pull request by mayakacz**
Signed-off-by: Maya Kaczorowski <15946341+mayakacz@users.noreply.github.com>

Continues from https://github.com/tailscale/tailscale/pull/6470
* Add LongHelp to all CLI commands that only had ShortHelp
* Uses feature functionality words and explanations consistent with how we're talking about the feature
* Clarifies that a key for support means _Tailscale_ support

### CODE REVIEW

**Comment by DentonGentry** [comment]
go has a canonical whitespace arrangement implemented by the `go fmt` tool, which we use. If you could:
- `go fmt`
- `git add cmd/tailscale/cli/network-lock.go`
- `git commit --amend`
- `git push -f`

It will update the whitespace in this commit.

File: cmd/tailscale/cli/network-lock.go [hunk 1]
```diff
@@ -359,7 +365,18 @@ func runNetworkLockSign(ctx context.Context, args []string) error {
 var nlDisableCmd = &ffcli.Command{
 	Name:       "disable",
 	ShortUsage: "disable <disablement-secret>",
-	ShortHelp:  "Consumes a disablement secret to shut down tailnet-lock across the tailnet",
+	ShortHelp:  "Consumes a disablement secret to shut down tailnet lock for the tailnet",
+	LongHelp: strings.TrimSpace(`
+
+The 'tailscale lock disable' command uses the specified disablement
+secret for tailnet lock.
```
Comment (twitchyliquid64) [inline cmd/tailscale/cli/network-lock.go:372]:
```suggestion
secret to disable tailnet lock.
```

File: cmd/tailscale/cli/network-lock.go [hunk 2]
```diff
@@ -377,7 +394,18 @@ func runNetworkLockDisable(ctx context.Context, args []string) error {
 var nlLocalDisableCmd = &ffcli.Command{
 	Name:       "local-disable",
 	ShortUsage: "local-disable",
-	ShortHelp:  "Disables the currently-active tailnet lock for this node",
+	ShortHelp:  "Disables tailnet lock (currently enabled) for this node only",
```
Comment (twitchyliquid64) [inline cmd/tailscale/cli/network-lock.go:397]: WRT `(currently enabled)` - this command help will always be visible regardless of whether tailnet lock is enabled or not. Do we still want to have that text?

**Review by twitchyliquid64 — APPROVED** [review]

File: cmd/tailscale/cli/network-lock.go (hunk 2 above)
Comment (mayakacz) [reply cmd/tailscale/cli/network-lock.go:383]: Ah, no, I didn't get that. Removing.

**Comment by mayakacz** [comment]
I know I'm doing the commits wrong. I don't know how to fix this because of the merge, and I am not going to attempt to understand git right now

**Comment by DentonGentry** [comment]
> I don't know how to fix this because of the merge

Just make sure to Squash and Merge when you submit this so it will turn into one commit in `main`, and we'll handle anything else in a subsequent pull request.

**Pull request merged**